### ISSUE TITLE
msvc builds are broken in the workflow: missing gettext

### ISSUE THREAD

**Issue by tothxa** [widelands/widelands#6383]
**Describe the bug**
msvc builds fail in the workflow, cmake fails because it doesn't find gettext, even though it seems to be installed in the dependency installation step. Both 32 and 64 bit builds are affected.

vcpkg dependencies are restored from 3 weeks old cache, last 2 attempts to update also failed in link step of test Widelands build due to missing wavpack symbols.

**To reproduce**
See:
https://github.com/widelands/widelands/actions/runs/8082106344/job/22087237388
https://github.com/widelands/widelands/actions/runs/8078369949/job/22077932922

vcpkg update failures:
https://github.com/widelands/widelands/actions/runs/8013830338/job/21891473555
https://github.com/widelands/widelands/actions/runs/7925133118/job/21637803152

**Comment by bunnybot** [comment]
Mirrored on Codeberg as [#CB4745](https://codeberg.org/wl/widelands/issues/4745).

**Comment by matthiakl** [comment]
Wavpack dependency will be changed to an optional feature soon (https://github.com/microsoft/vcpkg/pull/36982), so no changes should be required for this to work again. I tried to add it as dependency, but cmake does not find it for some reason (https://github.com/matthiakl/widelands/actions/runs/7943116820/job/21687120736).

For the msgmerge issue, it looks like the binary was removed in the latest git version, so it should be added to the vcpkg dependencies (`gettext[tools]`).
